src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,50 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+type Props = {
+  setFilter: (filter: string) => void;
+  query: string;
+  setQuery: (query: string) => void;
+};
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+export const TodoFilter: React.FC<Props> = ({ setFilter, query, setQuery }) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={event => setFilter(event.target.value)}
+          >
+            <option value="all">All</option>
+            <option value="active">Active</option>
+            <option value="completed">Completed</option>
```

Comment:
Ensure that the filter values ('all', 'active', 'completed') are consistent with the values used in the `App.tsx` file. This will prevent any mismatches in filtering logic.